data/block.go

```diff
@@ -91,33 +51,22 @@ type HyperblockApiResponsePayload struct {
 
 // Hyperblock contains all fully executed (both in source and in destination shards) transactions notarized in a given metablock
 type Hyperblock struct {
-	Nonce                  uint64             `json:"nonce"`
-	Round                  uint64             `json:"round"`
-	Hash                   string             `json:"hash"`
-	PrevBlockHash          string             `json:"prevBlockHash"`
-	Epoch                  uint32             `json:"epoch"`
-	NumTxs                 uint32             `json:"numTxs"`
-	ShardBlocks            []*NotarizedBlock  `json:"shardBlocks"`
-	Transactions           []*FullTransaction `json:"transactions"`
-	Timestamp              time.Duration      `json:"timestamp,omitempty"`
-	AccumulatedFees        string             `json:"accumulatedFees,omitempty"`
-	DeveloperFees          string             `json:"developerFees,omitempty"`
-	AccumulatedFeesInEpoch string             `json:"accumulatedFeesInEpoch,omitempty"`
-	DeveloperFeesInEpoch   string             `json:"developerFeesInEpoch,omitempty"`
-	EpochStartInfo         *EpochStartInfo    `json:"epochStartInfo,omitempty"`
-	Status                 string             `json:"status,omitempty"`
-}
-
-// EpochStartInfo is a structure that hold information about epoch start meta block
-type EpochStartInfo struct {
-	TotalSupply                      string `json:"totalSupply"`
-	TotalToDistribute                string `json:"totalToDistribute"`
-	TotalNewlyMinted                 string `json:"totalNewlyMinted"`
-	RewardsPerBlock                  string `json:"rewardsPerBlock"`
-	RewardsForProtocolSustainability string `json:"rewardsForProtocolSustainability"`
-	NodePrice                        string `json:"nodePrice"`
-	PrevEpochStartRound              uint64 `json:"prevEpochStartRound"`
-	PrevEpochStartHash               string `json:"prevEpochStartHash"`
+	Hash                   string                              `json:"hash"`
+	PrevBlockHash          string                              `json:"prevBlockHash"`
+	Nonce                  uint64                              `json:"nonce"`
+	Round                  uint64                              `json:"round"`
+	Epoch                  uint32                              `json:"epoch"`
+	NumTxs                 uint32                              `json:"numTxs"`
+	AccumulatedFees        string                              `json:"accumulatedFees,omitempty"`
+	DeveloperFees          string                              `json:"developerFees,omitempty"`
+	AccumulatedFeesInEpoch string                              `json:"accumulatedFeesInEpoch,omitempty"`
```

Comment:
please revert: don't change the json tags!! make sure the new tags are exactly the same as before

Reply:
structure is the same